doc/conf_ref.rst

```diff
@@ -222,6 +222,19 @@ or :ref:`mirrorlist <mirrorlist-label>` option definition.
     RPM debug scriptlet output level. One of: ``critical``, ``emergency``,
     ``error``, ``warn``, ``info`` or ``debug``. Default is ``info``.
 
+.. _search_enabled_metadata-label:
+
+``search_enabled_metadata``
```

Comment:
@ignatenkobrain @jsilhan The behavior introduce in this repository cannot be default. See description about risks for automatic scripts. But we can introduce new comman line option that will enable it.